lineage/tHMM.py

```diff
@@ -232,9 +229,13 @@ def fit_list(tHMMobj_list, tolerance=1e-9, max_iter=1000):
         old_LL = total_new_LL
 
         do_M_step_list(tHMMobj_list, MSD_list, betas_list, gammas_list)
-        for tHMM in tHMMobj_list:
+        for idx, tHMM in enumerate(tHMMobj_list):
             MSD, NF, betas, gammas = do_E_step(tHMM)
             new_LL.append(np.sum(calculate_log_likelihood(NF)))
```

Comment:
Do you ever empty the `new_LL` list for the next fitting iteration?

Reply:
Oh.. I guess not. Will do it right away. Thanks